kuttl-test.yaml

```diff
@@ -3,5 +3,5 @@ kind: TestSuite
 crdDir: ./operator/config/crd/bases
 testDirs:
   - ./test/integration/
-timeout: 400
+timeout: 500
```

Comment:
Is this only due to the big python-runtime image download ? If yes, let's revert it

Reply:
yeah apparently there is some other problem but I cannot reproduce it locally

Reply:
this test was passign when I used my docker image reference in the pipeline... I would maybe investigate that